### ISSUE TITLE
How to apply decontam to functional data

### ISSUE THREAD

**Issue by Ivy-ops** [benjjneb/decontam#151]
Hi Developer,
Thank you for developing this tool. Applying it to feature count data is quite straightforward. As you know functional information is also important for the microbiome data. I was wondering if it can also be applied to the HUMAnN3 output, which is CPM-transformed data?
Thanks!

**Comment by benjjneb** [comment]
Yes, if you have the appropriate auxiliary data. Either the same functional profiles from negative controls, or measurements of the input DNA concentrations.

**Comment by Ivy-ops** [comment]
@benjjneb Thank you! I saw the tutorial is on phyloseq data format. How can have a decontamed table? As my input is a OTU table, and my output wants to be a decontamed OTU table. 

The demo code only returns a list of vectors. 

```
st <- readRDS(system.file("extdata", "st.rds", package="decontam"))
conc <- c(6413, 3581.0, 5375, 4107, 4291, 4260, 4171, 2765, 33, 48)
neg <- c(FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, TRUE, TRUE)
isContaminant(st, conc=conc, method="frequency", threshold=0.2) 
```

**Comment by benjjneb** [comment]
`is.conc <- isContaminant(st, conc=conc, method="frequency", threshold=0.2)`

This will return a TRUE/FALSE vector (a logical vector) where TRUE values indicate contaminants identifed at the 0.2 threshold.

If you want to remove those OTUs/ASVs, then `st.nocontam <- st[,!is.conc]` will return a sequence table without the contaminant-classified OTUsASVs.
